crowdin.yml

```
@@ -0,0 +1,8 @@
+project_id: '512984'
```

```suggestion
project_id: ''
```

Empty for now until we know the actual ID after setting up the project